src/components/interaction-designer/BuilderCanvas.vue

```diff
@@ -186,7 +187,7 @@ export default class BuilderCanvas extends Vue {
       return this.windowWidth - this.widthAdjustment
     }
 
-    const xPosition: number = this.blockAtTheLowestPosition.ui_metadata.canvas_coordinates.x
+    const xPosition: number = this.blockAtTheFurthestRightPosition.ui_metadata.canvas_coordinates.x
```

Comment:
fixed wrong copy-paste from `canvasHeight()`

Reply:
Can we use this opportunity to start the first block from the left-top corner or center. Right now it starts from the bottom-right corner. Suggesting this as it feels like this is the right spot to make the change.
If it is a time consuming change, then we can handle it later.

Reply:
@poojahpatil90 , I'd like to handle that later in a different ticket.